Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -55,6 +55,14 @@ public float FlatHandThreshold
             set { flatHandThreshold = value; }
         }
 
+        [SerializeField]
+        [Tooltip("When using hybrid hand rotation, solver hand rotation until the menu is aligned with the gaze sufficiently")]
+        private bool useHybridHandRotation = false;
+
+        [SerializeField]
+        [Tooltip("Angle between hand up and camera forward, below which the hand menu follows the gaze, if useHybridHandRotation is active.")]
+        private float hybridHandRotationThresholdAngle = 60f;
```

Comment:
Thanks for these new behaviors! Please also [update the documentation](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/PullRequests.html#documentation-should-be-added-in-the-same-pull-request-as-a-fix--feature) to describe the new fields. The Solvers Page, section [Hand Menu with HandConstraint PalmUp](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Solver.html#hand-menu-with-handconstraint-and-handconstraintpalmup) would be a great place to describe the new behavior. 

Updating documentation along with the new behaviors helps ensure that the documentation for MRTK stays up to date.

Reply:
I just updated the docs!